frontend/src/metabase/modes/components/drill/ColumnFilterDrill.tsx

```diff
@@ -35,7 +45,7 @@ export default function ColumnFilterDrill({ question, clicked }) {
       popover: ({ onChangeCardAndRun, onResize, onClose }) => (
         <FilterPopover
           query={query}
-          filter={initialFilter}
+          filter={initialFilter as any}
```

Comment:
TypeScript warns that `initialFilter` can be `undefined`. Should we offer this drill if we can't figure out the filter?